x/cronos/types/events.go

```diff
@@ -9,12 +9,16 @@ import (
 const (
 	AttributeValueCategory = ModuleName
 
-	AttributeKeyRecipient = "recipient"
-	AttributeKeySender    = "sender"
+	AttributeKeyRecipient             = "recipient"
+	AttributeKeySender                = "sender"
+	AttributeKeyAmount                = "amount"
+	AttributeKeyReceiver              = "receiver"
+	AttributeKeyEthereumTokenContract = "ethereum_token_contract"
 
 	// events
-	EventTypeConvertVouchers = "convert_vouchers"
-	EventTypeTransferTokens  = "transfer_tokens"
+	EventTypeConvertVouchers             = "convert_vouchers"
+	EventTypeTransferTokens              = "transfer_tokens"
+	EventTypeEthereumSendToCosmosHandled = "ethereum_send_to_cosmos_handled"
```

Comment:
`ethereum_to_cronos_handled` to be more accurate?

or `ethereum_to_crc20_handled`

Reply:
Depends on whether we want a more generic event type or specific one. If the long run we want to push https://github.com/crypto-org-chain/gravity-bridge/pull/12 to upstream. Then it is better to stick with the Gravity contract event name.

Reply:
yeah but the handled do also gravity->crc20 conversion which is not exactly only sendToCosmos event

Reply:
or move the event firing at the beginning of the function

Reply:
updated.